package.json

```diff
@@ -165,6 +165,11 @@
       "yarn lint:fix"
     ]
   },
+  "rnpm": {
```

Comment:
isn't rnpm deprecated?
https://github.com/react-native-community/react-native-maps/issues/2945

Reply:
yup, actually fonts were already linked in `react-native.config.js` by me in other PR :laughing: